codice/ddf
# [2.13.x]DDF-04838 Clarify that new workspaces inherit search filters

#### What does this PR do?
Updates documentation to clarify Intrigue behavior.
#### Who is reviewing it? 
@brjeter  @austinsteffes @tyler30clemens 
#### Select relevant component teams: 
@codice/docs 
#### Ask 2 committers to review/merge the PR and tag them here.
@brjeter
@vinamartin

#### How should this be tested?
1. Build documentation
2. Open documentation.html
3. Inspect section "11.2.1. Creating a Workspace in Intrigue" for errors and fitness to purpose (see also attached screenshot)

#### What are the relevant tickets?
Fixes: #4838 

#### Screenshots
![image](https://user-images.githubusercontent.com/133284/57885608-2cd44800-77e0-11e9-86b8-4777974e3be3.png)


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
